pkgs/development/python-modules/moderngl/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, libGLU_combined
+, libX11
+}:
+
+buildPythonPackage rec {
+  pname = "moderngl";
+  version = "5.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0x8xblc3zybp7jw9cscpm4r5pmmilj9l4yi1rkxyf0y80kchlxq4";
+  };
+
+  disabled = !isPy3k;
+
+  buildInputs = [ libGLU_combined libX11 ];
```

Comment:
In https://aur.archlinux.org/packages/python-moderngl-git/, they only use `libGL`. What did you infer these dependencies from?

Reply:
Tried a build without `libX11` and it crashed with:

```In file included from src/GLContext.cpp:561:0:
/nix/store/34kliyybc28hydrxv4f7l0h9p6wm6dhp-libGLU-combined/include/GL/glx.h:30:10: fatal error: X11/Xlib.h: No such file or directory
 #include <X11/Xlib.h>
          ^~~~~~~~~~~~
compilation terminated.
error: command 'gcc' failed with exit status 1
```

Reply:
Alright, I get the same. However, you should replace `libGLU_combined` by `libGL` or is there a reason not to?

Reply:
And we should mark it as broken on Darwin as long as that doesn't work.

Reply:
Tested it with `libGL` and that works just fine. I've removed Darwin from the platforms, added a comment that it is broken on that platform.